lib/views/list.js

```diff
@@ -29,7 +30,9 @@ class List extends Component {
     };
 
     this.onItemLongPress = this.onItemLongPress.bind(this);
-    this.scanner = new Scanner(this.onItemFound.bind(this));
+    this.scanner = new Scanner(this.onItemFound.bind(this), () => {
+      Alert.alert('Network error', 'Please check your internet connection');
+    });
```

Comment:
NIT: Would prefer this as a named options object. Otherwise I don't know what the second argument is for without reading the implementation of `Scanner`.